build.gradle

```diff
@@ -77,11 +77,11 @@ dependencies {
   implementation "com.sonatype.nexus:nexus-platform-api:$nexusPlatformApiVersion"
   implementation "org.sonatype.ossindex:ossindex-service-client:$ossIndexClientVersion"
   implementation "ch.qos.logback:logback-classic:$logbackVersion"
+  implementation "commons-io:commons-io:$commonsIoVersion"
```

Comment:
Required to process `package-lock.json` (and possibly other manifests) files when scanning them.